builder/builder-next/builder.go

```diff
@@ -437,7 +445,7 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 
 	eg.Go(func() error {
 		for sr := range ch {
-			dt, err := sr.Marshal()
+			dt, err := proto.Marshal(sr) // FIXME(thaJeztah): is this the correct replacement?
```

Comment:
Don't know if this is the right replacement; if someone knows how to verify this?

Reply:
This is correct.

Reply:
Thanks! Let me remove this FIXME 👍